android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -473,11 +474,11 @@ public void onKeyboardDismissed() {
     resizeTextView(false);
   }
 
-  public void downloadKMP(String packageId, String bcp47) {
+  public void downloadKMP(String packageId, String bcp47, KmpInstallMode installMode) {
     Uri downloadUri = bcp47 == null ?
       Uri.parse(KMString.format("https://keyman.com/go/package/download/%s", new Object[]{packageId})) :
       Uri.parse(KMString.format("https://keyman.com/go/package/download/%s?bcp47=%s", new Object[]{packageId, bcp47}));
```

Comment:
I missed this on the last PR.
Do we want to use the paths `KMPLink.KMP_DOWNLOAD_KEYBOARDS_FORMATSTR` so we can also handle links from keyman-staging.com?

Hopefully that handles referrer too

Reply:
Given this is a stopgap function until we refactor downloadKMP one day in the future, I don't think we need to do this. The pattern here is used to give downloadKMP something it recognises; it then uses the package name and the bcp 47 id out of that string.